### DISCUSSION TITLE
Nuget in Visual Studio does not show all RavenDb.Client versions

### DISCUSSION THREAD

**Discussion by cadilhac** [ravendb/ravendb#19507] [Q&A] [answered]
In Visual Studio 2022, Nuget proposes only the latest stable version when RavenDb is not installed. If I manually install a specific version, like 5.4.203, then Nuget now shows only this specific version as available.
With 5.4.203 installed, the Updates tab shows no newer versions to update to.
I checked other packages and they all show all their available versions.

![nuget](https://github.com/user-attachments/assets/5f011976-a49f-4e37-8d5b-504f4e02f7a4)

**Comment by cadilhac** [comment]
I opened another project in VS 2019, and RavenDb versions appear correctly. So it seems to be an issue in VS 2022 only. I tried to delete the local nuget cache, delete .vs. Nothing works.

**Reply by ppekrol** [reply]
Hi, it took around 15s on my VS2022 to load the list there. All the packages are listed, if anything that would be VS issue.

**Comment by cadilhac** [comment]
Yes of course it's a VS issue. But I was wondering if someone already had this and found a way to fix it...

**Reply by ppekrol** [reply]
You have latest one? I know that NuGet is using CDN so maybe there s a problem in your region at the moment?

**Reply by cadilhac** [reply]
Yes, latest VS preview. Will check again later then,

**Reply by ppekrol** [reply]
Hmm, I checked my preview and I can confirm that I see only latest package there. Worth probably submitting to MS https://developercommunity.visualstudio.com/VisualStudio

**Reply by cadilhac** [reply]
Done

**Reply by ppekrol** [reply]
Can you paste the link here? We will vote for it.

**Reply by cadilhac** [reply]
https://developercommunity.visualstudio.com/t/In-VS2022-1712-preview-4-Nuget-does-no/10776126

**Reply by cadilhac** [reply, marked as answer]
They have fixed it in version 17.13.0 Preview 1.0.
